configure.ac

```diff
@@ -560,6 +561,12 @@ AC_ARG_ENABLE([util-wallet],
   [build_bitcoin_wallet=$enableval],
   [build_bitcoin_wallet=$build_bitcoin_utils])
 
+AC_ARG_ENABLE([util-util],
+  [AS_HELP_STRING([--enable-util-util],
```

Comment:
Why is this needed, since there's already --with-util above? (Also, why util-util?)

Reply:
`--with-utils` gives the bitcoin-cli, bitcoin-tx, bitcoin-util, bitcoin-wallet utils; `--with-util-X` just gives bitcoin-X. Not really needed, (and util-util is certainly weird) but don't see any reason to be inconsistent.